CMakeLists.txt

```diff
@@ -148,7 +148,7 @@ install(CODE "execute_process(COMMAND ./scripts/install_deps.sh
 # https://bitbucket.org/eigen/eigen/
 ExternalProject_Add(eigen
   PREFIX ${GraphLab_SOURCE_DIR}/deps/eigen
-  URL https://s3-us-west-2.amazonaws.com/static.graphlab.com/PowerGraph/deps/eigen_3.1.2.tar.bz2
+  URL https://s3.amazonaws.com/static.dato.com/PowerGraph/deps/eigen_3.1.2.tar.bz2 
```

Comment:
Why not use http://static.dato.com directly? That should be faster (CloudFront vs. S3 http access).